packages/@o3r/training-sdk/.openapi-generator-ignore

```diff
@@ -0,0 +1 @@
+# Indicate the index.ts file you have override to redirect to custom interface definition
```

Comment:
I am not sure what this is supposed to mean (should be fixed in generator as well)

Reply:
I agree this is not very clear, we should fix this in the generator and then re-generate this SDK when this update is implemented?